### PR TITLE
Issue #8: Create Dependencies File

### PR DESCRIPTION
**Pull request by jamacado**
**Create Dependencies File**: Issue[ #8](https://github.com/Architect-Coders/MU8/issues/8).

- Create file dependencies

**Testing Done:**

- On Xiaomi A3 API 28

**How it looks:**

- In Android Studio:

imagen

**How to test:**

- Run App

**Signed-off-by:**

    Javier M Cardona <javier.mcardona@gmail.com>

### CODE REVIEW

File: buildSrc/src/main/java/Dependencies.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    const val retrofit = "2.6.2"
+
+    //Java
+    const val javaCompatibility = "7"
+
+    //Detekt
+    const val detekt  = "1.1.1"
+}
+
+object Deps {
```
Comment (recaldev) [inline buildSrc/src/main/java/Dependencies.kt]: Please rename this as `GeneralDependencies` to improve understanding, also, please check if you can create different objects for the different types of dependencies.

For example see if you can create another object called `DataDependencies` and put there retrofit and glide.

File: data/build.gradle [hunk 2]
```diff
@@ -26,10 +26,13 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'androidx.core:core-ktx:1.1.0'
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+
```
Comment (recaldev) [inline data/build.gradle:26]: Remove extra space

File: build.gradle [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
+    ext.gradle_version = '3.5.2'
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath "com.android.tools.build:gradle:$gradle_version"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         // NOTE: Do not place your application dependencies here; they belong
```
Comment (recaldev) [inline build.gradle]: Remove this comments

File: buildSrc/build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import org.gradle.kotlin.dsl.`kotlin-dsl`
+
+plugins {
+    `kotlin-dsl`
+}
+repositories {
+    jcenter()
```
Comment (recaldev) [inline buildSrc/build.gradle.kts:7]: Is this really needed? Isn't enough with the `jcenter()` included on the project gradle?

**Review by recaldev — CHANGES_REQUESTED** [review]

File: buildSrc/build.gradle.kts (hunk 4 above)
Comment (jamacado) [reply buildSrc/build.gradle.kts:7]: when eliminating these lines, the following error appears when synchronizing "Could not find org.jetbrains.kotlin:kotlin-scripting-compiler-embeddable:1.3.21.
Required by: project :buildSrc"

File: buildSrc/src/main/java/Dependencies.kt (hunk 1 above)
Comment (jamacado) [reply buildSrc/src/main/java/Dependencies.kt]: the dependencies are separated into different objects

File: data/build.gradle (hunk 2 above)
Comment (jamacado) [reply data/build.gradle:26]: is removed extra space

**Review by recaldev — APPROVED** [review]

**Review by aliceresponde — APPROVED** [review]

**Pull request merged**